docs/source/models/concept_models/vivarium_mncnh_portfolio/facility_model.rst

```diff
@@ -0,0 +1,97 @@
+:orphan:
+
+.. _2024_facility_model_vivarium_mncnh_portfolio:
+
+Delivery Facility Choice Model
+==============================
+
+.. contents::
+   :local:
+   :depth: 2
+
+Background
+----------
+
+To capture the complex relationship between choice of delivery facility (home birth vs a facility with basic emergency obstetric and neonatal care [BEmONC] vs a facility comprehensive care [CEmONC]), the belief about gestational age (believed pre-term vs believed full term), and the related factors of antenatal care (ANC), and low birth weight small for gestational age (LBWSG) risk exposure, we will include two novel affordances in our simulation: (1) correlated propensities for ANC, home delivery, and LBWSG; and (2) conditional probabilities for home delivery that differ based on the believe pre-term/full-term status when labor begins.
+
+Coming up with values for these correlations and conditional probabilities that are consistent with GBD and external evidence is detailed at the end of this document.  But before we get to that complexity, let's start with how we will use these correlations and conditional probabilities in the simulation.
+
+[[Causal model diagram goes here]]
+
```

Comment:
Somewhere, I think we should mention that the calibration procedure, and hence the values we're using here (i.e., the correlations and values of P(facility | believed preterm status)) depend critically on the implementations of other pieces of the model that are described elsewhere (most notably, choice of ultrasound type given ANC status, and deriving an estimated gestational age from the true gestational age).